src/spikeinterface/comparison/groundtruthstudy.py

```diff
@@ -42,6 +42,11 @@ class GroundTruthStudy:
 
     This GroundTruthStudy have been refactor in version 0.100 to be more flexible than previous versions.
     Note that the underlying folder structure is not backward compatible!
+
+    Parameters
+    ----------
+    study_folder : srt | Path
```

Comment:
```suggestion
    study_folder : str | Path
```

Reply:
Thanks @chrishalcrow 

I'm actually working on the GTStudy these days so I'll fill in the rest of the missong docstrings!